.circleci/continue-config.yml

```diff
@@ -114,6 +114,15 @@ workflows:
                 root: '.'
                 paths:
                   - ./packages/*/dist # persist dist folder too
+            - run:
```

Comment:
🔴 this is great for PRs but we also need to set the baseline whenever we merge something to `master` like we do here for the `ui` library

https://github.com/voiceflow/design/blob/master/.circleci/continue_config.yml.tmpl#L139

so we need to add something similar to the `monorepo_release` job for this repo too

```
if [ "$CIRCLE_BRANCH" == "master" ]; then
  npx chromatic --project-token $CHROMATIC_PROJECT_TOKEN --build-script-name build:storybook --auto-accept-changes --exit-once-uploaded
else
  echo "Skipping Chromatic on non-master branch."
fi
```

🟢 I'd also recommend defining a `chromatic` script in `packages/react-chat/package.json` so that we can just call `yarn chromatic --project-token $CHROMATIC_PROJECT_TOKEN  [flags]` in this config and move the `--build-script-name build:storybook` to the `chromatic` script so we don't have to repeat it